src/sfctl/tests/custom_help_text_correctness_test.py

```diff
@@ -224,7 +224,7 @@ def test_custom_help_text(self):
             print()
             print(line)
 
-        allowable_lines_not_found = 74
+        allowable_lines_not_found = 77
```

Comment:
Has this been verified that the help text should deviate from swagger? If so, please ignore this comment. Thanks!